src/Proto.Actor/Supervision.cs

```diff
@@ -171,4 +172,51 @@ private bool RequestRestartPermission(RestartStatistics rs)
             return true;
         }
     }
+
+    public class ExponentialBackoffStrategy : ISupervisorStrategy
+    {
+        private readonly TimeSpan _backoffWindow;
+        private readonly TimeSpan _initialBackoff;
+        private readonly Random _random = new Random();
+
+        public ExponentialBackoffStrategy(TimeSpan backoffWindow, TimeSpan initialBackoff)
+        {
+            _backoffWindow = backoffWindow;
+            _initialBackoff = initialBackoff;
+        }
+
+        private long ToNanoseconds(TimeSpan timeSpan)
+        {
+            return Convert.ToInt64(timeSpan.TotalMilliseconds * 1000000);
+        }
+
+        private long ToMilliseconds(long nanoseconds)
+        {
+            return nanoseconds / 1000000;
+        }
+
+        public void HandleFailure(ISupervisor supervisor, PID child, RestartStatistics rs, Exception reason)
+        {
+            SetFailureCount(rs);
+            var backoff = rs.FailureCount * ToNanoseconds(_initialBackoff);
+            var noise = _random.Next(500);
+            var duration = TimeSpan.FromMilliseconds(ToMilliseconds(backoff + noise));
+            Task.Delay(duration).ContinueWith(t =>

```

Comment:
Probably should only continue on success with this.  Maybe continue with a log when an error occurs